pages/common/zpool.md

```diff
@@ -4,30 +4,28 @@
 
 - Show the configuration and status of all ZFS zpools:
 
-`zpool status [{{poolname}}]`
+`zpool status`
 
 - Check a ZFS pool for errors (verifies the checksum of EVERY block). Very CPU and disk intensive:
 
-`zpool scrub {{poolname}}`
+`zpool scrub {{pool_name}}`
 
 - List zpools available for import:
 
 `zpool import`
 
-- Import a zpool, optionally specifying a new name:
+- Import a zpool (optionally renaming it):
 
-`zpool import {{poolname}}`
-`zpool import {{poolname}} {{newpoolname}}`
+`zpool import {{pool_name}} {{[new_pool_name]}}`
```

Comment:
I think there isn't a strong justification for keeping the optional argument in this particular case, but we can reconsider this policy if an example eventually appears where it makes sense.